src/math/p5.Vector.js

```diff
@@ -365,76 +342,100 @@ p5.Vector = class {
     return this;
   }
 
+
+  calculateRemainder2D (xComponent, yComponent) {
+    if (xComponent !== 0) {
+      this.x = this.x % xComponent;
+    }
+    if (yComponent !== 0) {
+      this.y = this.y % yComponent;
+    }
+    return this;
+  }
+
+  /// HELPERS FOR REMAINDER METHOD
+  calculateRemainder3D (xComponent, yComponent, zComponent) {
+    if (xComponent !== 0) {
+      this.x = this.x % xComponent;
+    }
+    if (yComponent !== 0) {
+      this.y = this.y % yComponent;
+    }
+    if (zComponent !== 0) {
+      this.z = this.z % zComponent;
+    }
+    return this;
+  }
+
```

Comment:
I'm not sure how well it will work or be supported but I wonder can these be marked as [private methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties)?

Reply:
I am not sure the support of `#field` , so I use `@private`.